src/components/common/InfoSection.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import Iphone6 from 'assets/images/i6.svg';
+import AppStoreIcon from 'assets/images/appstore.svg';
+import SocialIcon from 'assets/images/social.svg';
```

Comment:
WDYT about creating an index file inside images where You can export all the svgs and then import them from any component as named imports?